Gopkg.toml

```diff
@@ -27,12 +27,23 @@
 
 
 required = [
+  "github.com/Azure/go-autorest/autorest",
+  "github.com/docker/distribution/reference",
   "k8s.io/api/core/v1",
+  "k8s.io/apiserver/pkg/endpoints/request",
```

Comment:
I needed to pin `k8s.io/apiserver` to 1.9.3 because in master request.Context has been updated to use the Go context.Context package.

https://github.com/kubernetes/apiserver/commit/cd0258b4d710b6e576193f188f797acca739504b